includes/class-wc-stripe-webhook-state.php

```diff
@@ -269,34 +298,40 @@ public static function get_webhook_status_message() {
 						 * translators: 1) date and time of last failed webhook e.g. 2020-06-28 10:30:50 UTC
 						 * translators: 2) reason webhook failed
 						 * translators: 3) date and time of last successful webhook e.g. 2020-05-28 10:30:50 UTC
+						 * translators: 4) number of pending webhooks
 						 */
-						__( 'Warning: The most recent test webhook, received at %1$s, could not be processed. Reason: %2$s. (The last test webhook to process successfully was timestamped %3$s.)', 'woocommerce-gateway-stripe' ) :
+						__( 'Warning: The most recent test webhook, received at %1$s, could not be processed. Reason: %2$s. (The last test webhook to process successfully was timestamped %3$s, and there are approximately %4$d webhooks pending)', 'woocommerce-gateway-stripe' ) :
```

Comment:
As written, this approach results in a big block of hard-to-parse text, and has edge cases with weird wording for the warning, e.g. `and there are approximately 1 webhooks pending`. I think it may be clearer to split the wording into a new separate sentence that uses `_n()` and is only shown when `$pending_webhooks > 0`.

Also as a much smaller nit, we are missing punctuation before the closing `)` in the new message.
```suggestion
						__( 'Warning: The most recent test webhook, received at %1$s, could not be processed. Reason: %2$s. (The last test webhook to process successfully was timestamped %3$s.)', 'woocommerce-gateway-stripe' ) :
```

And then later on we have code that adds something like:
```php
					. ( $pending_webhooks < 1 ? '' :
						' ' . _n(
							'There is at least %d webhook pending.',
							'There are approximately %d webhooks pending.',
							$pending_webhooks
						);